sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -171,222 +186,264 @@ Sample code snippets are provided to illustrate using long-running operations [b
 
 The following section provides several code snippets covering some of the most common Form Recognizer tasks, including:
 
-* [Recognize Forms Using a Custom Model](#recognize-forms-using-a-custom-model "Recognize Forms Using a Custom Model")
-* [Recognize Content](#recognize-content "Recognize Content")
+* [Extract layout](#extract-layout "Extract Layout")
```

Comment:
Analyze Layout ?

Reply:
Still prefer to leave this as "extract layout" for OCR and "analyze" for built/prebuilt analysis. Thoughts? @maririos @witemple-msft @catalinaperalta

Reply:
+1 to the distinction. Also looking at the [comments from Paul](https://github.com/Azure/azure-sdk-for-python/commit/191a9c9d1421cc2b271a078592770c60b3bb6577) I will add `Analyze` to `Document`

Reply:
Would it feel awkward to use `Extract` for an API that uses the verbiage `begin**Analyze**Document ?
But I am not too strong on it.

Reply:
Going to stick with this for beta.1. we can reevaluate naming again later.